.github/workflows/ci.yml

```diff
@@ -6,11 +6,14 @@ on:
 jobs:
   ci:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16, 18]
```

Comment:
❓ Why do we want to run the CI pipeline with node 16 and 18 when we use node 18 in development and we do not even publish a node package but a react component library?

Reply:
I'm just going to remove this. I'd like to get this merged in before i'm off.